data/json/items/comestibles.json

```diff
@@ -825,6 +825,7 @@
         "volume" : 1,
         "phase" : "liquid",
         "charges" : 5,
+        "flags": [ "EATEN_COLD" ],
```

Comment:
JSON style, please `flags` as last field to reduce future changes required if we start applying the JSON lint tool more